src/main/java/racingcar/domain/Judgement.java

```diff
@@ -0,0 +1,66 @@
+package racingcar.domain;
+
+import java.util.List;
+
+import static racingcar.common.SystemMessage.FINAL_WINNERS_INFO;
+import static racingcar.common.SystemMessage.NO_WINNER_INFO;
+
+public class Judgement {
+
+    private static final int ZERO = 0;
+    private static final String STEP = "-";
+    private static final String COMMA = ",";
+
+    private StringBuilder stringBuilder = new StringBuilder();
+
+    private Judgement() {
+    }
+
+    public static Judgement instance() {
+        return new Judgement();
+    }
+
+    public String judge(final List<Car> cars) {
+        stringBuilder.setLength(ZERO);
+        int max = maxDriveLength(cars);
+
+        if (max == ZERO) {
+            stringBuilder.append(NO_WINNER_INFO);
+            return stringBuilder.toString();
+        }
+
+        stringBuilder.append(FINAL_WINNERS_INFO);
+
+        for (Car car : cars) {
+            if (max == car.getStep()) {
+                stringBuilder.append(car.getName());
+                stringBuilder.append(COMMA);
+            }
+        }
+        stringBuilder.deleteCharAt(stringBuilder.lastIndexOf(COMMA));
+
+        return stringBuilder.toString();
+    }
+
+    private int maxDriveLength(final List<Car> cars) {
+        return cars.stream()
+                .mapToInt(Car::getStep)
+                .max().
+                getAsInt();
```

Comment:
```suggestion
                .max()
                .getAsInt();
```
`.`이 아래로 내려와야할 것 같습니다.

Reply:
줄바꿈하면서 잘못했나보네요ㅠ__ㅠ 감사합니다!